prow/ANNOUNCEMENTS.md

```diff
@@ -2,7 +2,10 @@
 
 New features added to each component:
 
- - *January 12, 2018* `blunderbluss` plugin now provides a new command, `/auto-cc`,
+ - *January 15, 2019* `approve` now considers self-approval and github review
+   state by default. Configure with `require_self_approval` and
+   `ignore_review_state`. Temporarily revert to old defaults with `use_deprecated_2017_implicit_self_approve_default_migrate_before_july_2018` and `use_deprecated_2017_review_acts_as_approve_default_migrate_before_july_2018`.
```

Comment:
july 2019?

Reply:
When I see `require_self_approval` it makes me think that this would require the PR author to `/approve` even if another approve explicitly `/approve`s. Is this what it means? If not, that might be a confusing thing to call this flag.

Reply:
Kath: done

Misty: Yes, when this is on, the files changed in my PR will not be approved by me until I write an `/approve` comment.